bevy_lint/README.md

```diff
@@ -0,0 +1,135 @@
+<div class = "rustdoc-hidden">
+
+# `bevy_lint`
+
+`bevy_lint` is a custom linter for the [Bevy game engine](https://bevyengine.org), similar to [Clippy](https://doc.rust-lang.org/stable/clippy).
+
+</div>
+
+<div class="rustdoc-alert rustdoc-alert-warning">
+
+> **Warning**
+>
+> This is an unofficial community project, hacked upon by the Bevy CLI working group until it is eventually upstreamed into the main [Bevy Engine organization]. Pardon our rough edges!
+
+</div>
+
+[Bevy Engine organization]: https://github.com/bevyengine
+
+## Installation
+
+`bevy_lint` uses [`#![feature(rustc_private)]`](https://doc.rust-lang.org/nightly/unstable-book/language-features/rustc-private.html) to link to `rustc` crates. As such, it requires a specific nightly toolchain to be installed.
```

Comment:
This would really benefit from an explanation / justification of why this is necessary.